test/extensions/filters/http/lua/BUILD

```diff
@@ -2,15 +2,19 @@ licenses(["notice"])  # Apache 2
 
 load(
     "//bazel:envoy_build_system.bzl",
-    "envoy_cc_test",
     "envoy_package",
 )
+load(
+    "//test/extensions:extensions_build_system.bzl",
+    "envoy_extension_cc_test",
+)
 
 envoy_package()
 
-envoy_cc_test(
+envoy_extension_cc_test(
     name = "lua_filter_test",
     srcs = ["lua_filter_test.cc"],
+    extension_name = "envoy.filters.http.lua",
```

Comment:
as an aside, buildifier seems to insist on having this on this line, vs. under "name" which seems a lot nicer.

Reply:
I think Buildifer is very opinionated about relative ordering of things like name/srcs, so I don't know if we can fix this.